scheduler/bin/help-make-cluster

```diff
@@ -18,8 +18,6 @@ CLUSTERNAME=$3
 COOK_KUBECONFIG=$4
 GKE_CLUSTER_OWNER=${GKE_CLUSTER_OWNER:-$USER}
 
-VERSION=1.15.11-gke.17
```

Comment:
No need to hardcode this. The default version is late enough, and now we don't need to refresh this to a later version every couple of months.